language/README.md

```diff
@@ -31,30 +31,30 @@ Learning API.
 
 View the [documentation][analyze_docs] or the [source code][analyze_code].
 
-__Usage:__ `node analyze --help`
+__Usage:__ `node analyze.js --help`
 
 ```
 Commands:
-  sentimentOfText <text>               Detect the sentiment of a block of text.
```

Comment:
The program doesn't throw an error if GOOGLE_APPLICATION_CREDENTIALS is not set. It silently swallows it and return nothing. We should fix this behavior.